spec/Section 5 -- Validation.md

```diff
@@ -198,6 +198,67 @@ query getName {
 }
 ```
 
+### Subscription Operation Definitions
+
+#### Single root field
+
+**Formal Specification**
+
+  * For each subscription operation definition {subscription} in the document
+  * Let {rootFields} be the top level selection set on {subscription}.
+    * {rootFields} must be a set of one.
+
+**Explanatory Text**
+
+Subscription operations must have exactly one root field.
```

Comment:
@stubailo i was agreeing that the original limitation of 1 subscription field makes sense at this phase until things will become more stable, just saying that it shouldn't be part of the spec since there is no real reason for that other then implementation.
(I'm already experimenting with subscriptions that has more then 1 event)

if that's not the case please enlighten me, i'm waiting for an answer for a week now..